lib/routes/charge/pos_invoice.dart

```diff
@@ -294,6 +300,12 @@ class POSInvoiceState extends State<POSInvoice> {
     );
   }
 
+  _onOnFocusNodeEvent() {
+    setState(() {
+      _isButtonDisabled = true;
```

Comment:
When the `_isButtonDisabled` going back to false?

Reply:
Whenever you click outside the keyboard.
```
Scaffold(
      resizeToAvoidBottomPadding: false,
      body: new GestureDetector(
        onTap: () {
          // call this method here to hide soft keyboard
          FocusScope.of(context).requestFocus(new FocusNode());
          setState(() {
            _isButtonDisabled = false;
          });
        },
```